5.0/config/log4j2.properties

```diff
@@ -0,0 +1,74 @@
+status = error
```

Comment:
Judging by the `log4j` file provided by elastic co in [their image](https://github.com/elastic/elasticsearch-docker/blob/9cf33f321c2d345b55786594c1c639bacb04432f/build/elasticsearch/log4j2.properties) I think this could be simplified.